dist/github-markdown.css

```diff
@@ -0,0 +1,1087 @@
+/* Generated from 'node_modules\github-markdown-css\github-markdown.css' */
+
+.markdown-body.markdown-light,
```

Comment:
Should this be checked in since it is generated? (I know the previous generated file was committed but think was probably a mistake)

Reply:
Checked out that file in b2c8012 👍🏾